index.html

```diff
@@ -2,16 +2,71 @@
 <html>
 <head>
   <meta charset="utf-8">
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
   <title>Self Directed FD Starter</title>
 </head>
 <body>
-  <h3>hello world</h3>
-      <!--  the a tag below can link you to another HTML page  -->
-      <!--  if you click the button in the browser right now, you'll get an error because that is not a real file path -->
-      <a href="<file path"><button class="default-button">GO TO OTHER PAGE</button></a>
-    <script
+  <header class="header">
+    <p class="title">playPlay</p>
+    <div class="artist-search">
+      <input type="text" name="artist" value="artist">
+      <button id="find-song-btn">Find Songs</button>
+    </div>
+  </header>
+  <section class="main">
+    <div class="container">
+      <div class="song1">
+        <p>song title</p>
+        <p>song genre</p>
+        <button id="favorite-song">
+          Favorite
+          <i class="far fa-star"></i>
+        </button>
+      </div>
+      <div class="song1">
+        <p>song title</p>
```

Comment:
I would change the name of the class to be something more general. The convention is that class names are applied to multiple elements but id's are unique.

So like just `song` 

Something we could do in the future is select the parent (`container` in this case) and target the `n`th child